starlette/applications.py

```diff
@@ -157,7 +157,23 @@ def host(
 
         self.router.host(host, app=app, name=name)
 
-    def add_middleware(self, middleware_class: type, **options: typing.Any) -> None:
+    def add_middleware(
+        self, middleware_class: type, **options: typing.Any
+    ) -> None:  # pragma: no cover
+        """
```

Comment:
I'll have to double check if we can add this doctoring.
As mentioned by @Kludex some of these methods are directly exposed through FastAPI and it was decided to not add more work for them now.
So we only add the docstrings to the ones that aren't exposed directly.

Reply:
Oh okay, that makes sense, I can remove it. Sorry if I missed the earlier convo.

Reply:
No worries. Thank you.